src/App.tsx

```diff
@@ -1,14 +1,29 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const [selectedOption, setSelectedOption] = useState('All');
```

Comment:
```suggestion
  const [selectedOption, setSelectedOption] = useState(Filter.All);
```
You should use enum values everywhere, so you have to import/export it
You might create a separate file types.ts and write this enum type there